contracts/eosio.system/src/delegate_bandwidth.cpp

```diff
@@ -225,7 +225,7 @@ namespace eosiosystem {
    void validate_b1_vesting( int64_t stake ) {
       const int64_t base_time = 1527811200; /// 2018-06-01
       const int64_t max_claimable = 100'000'000'0000ll;
-      const int64_t claimable = int64_t(max_claimable * double(now()-base_time) / (10*seconds_per_year) );
+      const int64_t claimable = int64_t(max_claimable * double(current_time_point().time_since_epoch().count()/1000000 - base_time) / (10*seconds_per_year) );
```

Comment:
Change `current_time_point().time_since_epoch().count()/1000000` to `current_time_point().sec_since_epoch()`